src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -22,19 +76,25 @@ export const TodoFilter: React.FC = () => {
           type="text"
           className="input"
           placeholder="Search..."
+          value={query}
+          onChange={handleInputChange}
         />
         <span className="icon is-left">
           <i className="fas fa-magnifying-glass" />
         </span>
 
-        <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-          <button
-            data-cy="clearSearchButton"
-            type="button"
-            className="delete"
-          />
-        </span>
+        {resetIsClicked && query
```

Comment:
do you really need the `resetIsClicked` variable? it would be better just to check the `query` length
```suggestion
        {query.length > 0
```